client/my-sites/stats/stats-module/all-time-nav.jsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ **/
+import React from 'react';
+import { connect } from 'react-redux';
+import { flowRight } from 'lodash';
+import { localize, moment } from 'i18n-calypso';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import SelectDropdown from 'components/select-dropdown';
+import { recordGoogleEvent } from 'state/analytics/actions';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSiteSlug } from 'state/sites/selectors';
+
+export const StatsModuleSummaryLinks = props => {
+	const { translate, path, siteSlug, query } = props;
+
+	const now = moment();
+	const quarter = moment().startOf( 'quarter' );
+	const daysSinceQuarterStart = now.diff( quarter, 'd' );
+	const options = [
+		{ value: '0', label: translate( 'Summary' ) },
+		{ value: '7', label: translate( 'Last 7 days' ) },
+		{ value: '30', label: translate( 'Last 30 days' ) },
+		{ value: `${ daysSinceQuarterStart }`, label: translate( 'the past quarter' ) },
+		{ value: '-1', label: translate( 'All Time' ) }
+	];
+
+	const onSelect = ( item ) => {
+		if ( item.value !== '0' ) {
```

Comment:
Since there is no path for when the value is `'0'`, selecting `Summary` in the dropdown does nothing. It feels weird, as is. I'm not sure if we should disable selecting it or what it should actually do if it's left enabled :)

<img width="256" alt="screen shot 2017-01-17 at 10 44 39 am" src="https://cloud.githubusercontent.com/assets/1587282/22027498/087dc7aa-dca2-11e6-826a-574c3fed82ac.png">